project/First-Project/src/main/java/first/project/acessingdatamysql/AuthenticationController.java

```diff
@@ -0,0 +1,53 @@
+package first.project.acessingdatamysql;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.List;
+
+@Controller
+public class AuthenticationController {
+
+    @Autowired
+    private UserRepository userRepo;
+
+    @GetMapping("/register")
+    public String Register(Model model) {
+        model.addAttribute("user", new User());
+
+        return "register";
+    }
+
+    @GetMapping("/login")
+    public String Login(Model model) {
+        model.addAttribute("user", new User());
+
+        return "login";
+    }
+
+    @PostMapping("/register sucessfull")
+    public String processRegister(User user)
+    {
+        List<User> list = userRepo.findAll();
```

Comment:
Instead of doing this much unnecessary work that is harder to understand, why simply not do this:

User u = userRepo.findByUserNameAndPassword(user.getEmail(), user.getPassword());
and if u is null than you will be able to save it in DB, else throw a message that, that user already exists.